include/flatbuffers/flatbuffers.h

```diff
@@ -1294,9 +1300,30 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     return true;
   }
 
+  // Returns the message size in bytes
+  size_t GetSize() const {
```

Comment:
Why `Internal`? `Computed` or `Calculated` seems more obvious where this comes from. And `GetBuffer` doesn't need to be renamed I think.